connectors/websocket/pom.xml

```diff
@@ -27,10 +27,10 @@
     <version>1.2.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>edgent-connectors-websocket-wsclient</artifactId>
```

Comment:
Ok ... it seems that there is a required dependency to "direct" provider, but I moved the dependency from a "compile" to a "test" dependency as removing "direct" completely caused compilation errors in tests.